autobuild/android/install-qt.sh

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env bash
```

Comment:
Not blocking, but maybe one time we could move over to aqt.

Reply:
aqt for all, but later, one after the overall structure

Reply:
Ok. Let's resolve this and work on it after this got merged.